alpine.Dockerfile

```diff
@@ -14,8 +14,6 @@ WORKDIR /ifc-$IFC_VERSION/
 COPY CMakeUserPresets.json .
 
 RUN cmake --preset=dev  # configure
-# change DESTINATION to bin/ so we don't need to modify PATH var
-RUN sed -i 's/TARGETS ifc-printer DESTINATION . COMPONENT Printer/TARGETS ifc-printer DESTINATION bin COMPONENT Printer/' /ifc-$IFC_VERSION/cmake/install-rules.cmake
```

Comment:
Note that this wasn't necessary to begin with. Everything is assigned to a component properly, so you can just install the components separately:

    cmake --install build --config Release --prefix <prefix> --component Development
    cmake --install build --config Release --prefix <prefix>/bin --component Printer

The above would've produced the result you desire.

Reply:
That's very useful. I have not used `cmake` for a while. Had to brush up on it to be able to do my humble contribution to ifc. Thanks again for the tip.